packages/yourloops/test/integration/assert/chat-widget.ts

```diff
@@ -25,25 +25,35 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { fireEvent, screen, waitFor, within } from '@testing-library/react'
-import { myFirstTeamName, mySecondTeamId, mySecondTeamName } from '../mock/team.api.mock'
+import { fireEvent, screen, within } from '@testing-library/react'
+import { myFirstTeamName, mySecondTeamName } from '../mock/team.api.mock'
 import userEvent from '@testing-library/user-event'
 import { monitoredPatientId } from '../data/patient.api.data'
 import ChatApi from '../../../lib/chat/chat.api'
 
-export const checkChatWidgetForPatient = async (): Promise<void> => {
+export const checkChatWidgetMessageReadingHcp = async (): Promise<void> => {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏It would be better to always keep the same convention for the method naming. I saw that we often have `doThisForPatient()`, `doThisForHcp()` etc so I think it would be nice to keep this convention here:
- `checkChatWidgetMessageReadingHcp` > `checkChatWidgetMessageReadingForHcp`
- `checkChatWidgetMessageReadingPatient` > `checkChatWidgetMessageReadingForPatient`